static/app/views/performance/transactionSummary/transactionOverview/durationChart/content.tsx

```diff
@@ -56,6 +60,44 @@ function Content({
     );
   }
 
+  const colors = (data && theme.charts.getColorPalette(data.length - 2)) || [];
+
+  // Create a list of series based on the order of the fields,
+  // We need to flip it at the end to ensure the series stack right.
+  const series = data
+    ? data
+        .map((values, i: number) => {
+          return {
+            ...values,
+            color: colors[i],
+            lineStyle: {
+              opacity: 0,
+            },
+          };
+        })
+        .reverse()
+    : [];
+
+  // Finds the range of yAxis values based on what is selected in the legend
+  let range;
+  if (series[1]) {
+    let minSeries = series[0];
+    let maxSeries;
+    series.forEach(({seriesName}, idx) => {
+      if (legend?.selected?.[seriesName] !== false) {
+        minSeries = series[idx];
+        maxSeries ??= series[idx];
+      }
+    });
+    const max = Math.max(...maxSeries?.data.map(({value}) => value));
+    const min = Math.min(
+      ...minSeries?.data.map(({value}) => value).filter(value => !!value)
+    );
+    range = {max, min};
+  }
```

Comment:
I would suggest moving this into a util function to detect bounds. It would remove the need to conditionally define that var, let you remove the comment (since you could just pick an explanatory name), and would also improve readability here a bit.